indico/web/forms/widgets.py

```diff
@@ -288,3 +288,13 @@ def __init__(self, show_field=True):
 
     def __call__(self, field, **kwargs):
         return super().__call__(field, input_args=kwargs)
+
+
+class PrefixedTextWidget(JinjaWidget):
+    """Render a TextInput with a prefix"""
+
+    def __init__(self, prefix=None, show_field=True):
+        super().__init__('forms/prefixed_text_widget.html', single_line=True, show_field=show_field, prefix=prefix)
+
+    def __call__(self, field, **kwargs):
+        return super().__call__(field, input_args=kwargs)
```

Comment:
I don't think this is needed. Have a look at the `single_kwargs` option of JinjaWidget though.

Reply:
Yes, I think I got led by the previous field.